Common/UnitDefinitions/ReciprocalLength.json

```diff
@@ -0,0 +1,131 @@
+{
+  "Name": "ReciprocalLength",
+  "BaseUnit": "InverseMeter",
+  "XmlDoc": "Reciprocal (Inverse) Length is used in various fields of science and mathematics. It is defined as the inverse value of a length unit.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Reciprocal_length",
+  "BaseDimensions": {
+    "L": -1
+  },
+  "Units": [
+    {
+      "SingularName": "InverseMeter",
+      "PluralName": "InverseMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "m⁻¹", "1/m" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "InverseCentimeter",
+      "PluralName": "InverseCentimeter",
+      "FromUnitToBaseFunc": "x*1e2",
+      "FromBaseToUnitFunc": "x/1e2",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cm⁻¹", "1/cm" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "InverseMillimeter",
+      "PluralName": "InverseMillimeter",
+      "FromUnitToBaseFunc": "x*1e3",
+      "FromBaseToUnitFunc": "x/1e3",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "mm⁻¹", "1/mm" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "InverseMile",
+      "PluralName": "InverseMile",
+      "FromUnitToBaseFunc": "x/1609.34",
+      "FromBaseToUnitFunc": "x*1609.34",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "mi⁻¹", "1/mi" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "InverseYard",
```

Comment:
I'm not familiar with the use of reciprocal length, so at first I was thinking, do we really need _all_ these units? But at least this online converter has similar units so I guess they are used:

https://www.unitsconverters.com/en/Reciprocal-Length-Conversions/Measurement-1182

Reply:
The reciprocal length quantity has been used in materials science and fluid dynamics. I've just implemented the major units and I think that these units would be sufficient for the moment. I try to keep as minimal as possible.